modules/Lending.py

```diff
@@ -171,9 +171,9 @@ def lend_all():
         lending_balances = Data.get_on_order_balances()
 
     # Fill the (maxToLend) balances on the botlog.json for display it on the web
-    for key in sorted(total_lended):
-        if len(lending_balances) == 0 or key not in lending_balances:
-            MaxToLend.amount_to_lend(total_lended[key], key, 0, 0)
+    for cur in sorted(total_lended):
```

Comment:
@laxdog even though you are 100% right to change this name, I would avoid **next time** doing naming changes as part of such a big change.